integration/chaincode/multi/chaincode.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package multi
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/hyperledger/fabric-chaincode-go/v2/shim"
+	pb "github.com/hyperledger/fabric-protos-go-apiv2/peer"
+)
+
+type Operations struct{}
+
+// Init initializes chaincode
+// ===========================
+func (t *Operations) Init(_ shim.ChaincodeStubInterface) *pb.Response {
+	return shim.Success(nil)
+}
+
+// Invoke - Our entry point for Invocations
+// ========================================
+func (t *Operations) Invoke(stub shim.ChaincodeStubInterface) *pb.Response {
+	function, args := stub.GetFunctionAndParameters()
+	fmt.Println("invoke is running " + function)
+
+	startWB, _ := strconv.ParseBool(args[0])
+	if startWB {
+		stub.StartWriteBatch()
+	}
+
+	switch function {
+	case "invoke":
+		return t.put(stub, args[1:])
```

Comment:
Only a single argument is passed right? e.g. `args[1]` , better to use a variable with an expressive name to make it clear what exactly is being passed here.

Reply:
done